frontend/src/components/ViewTrips/trips-container.js

```diff
@@ -18,13 +18,13 @@ const db = app.firestore();
 function getErrorElement(error) {
   return new Promise(function(resolve) {
     console.log(`Error in Trips Container: ${error}`);
-    resolve(
+    resolve((
       <div>
         <p>Oops, it looks like we were unable to load your trips.
                       Please wait a few minutes and try again.
         </p>
       </div>
-    );
+  ));
```

Comment:
Why the double parentheses (green highlighted change)?

Reply:
Whoops this was an accident. Must have done this when I ran into a bug of not calling this function with async. Changed it back.